spec.html

```diff
@@ -29747,11 +29743,12 @@ <h1>String.prototype.lastIndexOf ( _searchString_ [ , _position_ ] )</h1>
           1. Let _searchStr_ be ? ToString(_searchString_).
           1. Let _numPos_ be ? ToNumber(_position_).
           1. Assert: If _position_ is *undefined*, then _numPos_ is *NaN*.
-          1. If _numPos_ is *NaN*, let _pos_ be *+&infin;*; otherwise, let _pos_ be ! ToInteger(_numPos_).
+          1. If _numPos_ is *NaN*, let _pos_ be +&infin;; otherwise, let _pos_ be ! ToIntegerOrInfinity(_numPos_).
           1. Let _len_ be the length of _S_.
-          1. Let _start_ be min(max(_pos_, 0), _len_).
+          1. Let _start_ be the result of clamping _pos_ between 0 and _len_.
```

Comment:
does "the length of" always produce a mathematical value?

Reply:
Yes, for the same reason that "how many apples are on the table?" always produces a mathematical value.